scripts/batch-png-to-svg.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Right now this method only works with DE's simple icon PNGs so I wouldn't try this with
+# anything more complex then an icon. Even then the original would have to be something >= 256x256
+# otherwise you could have misalinged lines and basically blobs
+
+# Loop through PNG files
+for file in *.png; do
+  # Extract filename without extension
+  basename=$(echo "$file" | sed 's/\.\w*$//')
+
+ # Convert PNG to black PNG with transparency
+ # We need to convert the image from black to white so potrace can detect
+ # and trace properly
```

Comment:
needs another space at the front

```suggestion
  # Convert PNG to black PNG with transparency
  # We need to convert the image from black to white so potrace can detect
  # and trace properly
```

Reply:
Wouldn't worry to much about this script, I'll be actively looking for alternatives